### PR TITLE
PageHeader and  ErrorBox added to /dist folder

### PR DESCRIPTION
**Pull request by philjonasfiltered**
# <JIRA issue / Title>

## TL;DR

-  the title says it all. Only relevant files exported.

## Details

Checklist / Reminders:

-   [ ] Documentation / Story is created / adapted
-   [ ] Unit tests
-   [ ] CSS changes are BEM-compliant

### CODE REVIEW

File: dist/components/Organisms/PageHeader/PageHeader.d.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import { PageHeaderProps } from './PageHeader.interface';
+export declare const PageHeader: React.FC<PageHeaderProps>;
+//# sourceMappingURL=PageHeader.d.ts.map
```
Comment (tricaalexandru) [inline dist/components/Organisms/PageHeader/PageHeader.d.ts:4]: @philjonasfiltered do we need this commented out line?

File: dist/components/Organisms/PageHeader/PageHeader.interface.d.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+export interface PageHeaderProps {
+    imageURL: string;
+    languageOptions: DropdownOptionProps[];
+    onLogoClick?: (e: React.MouseEvent<HTMLElement>) => void;
+    onLanguageChange?: (index: number) => void;
+    onboardingClick?: (ev: React.MouseEvent<HTMLElement>) => void;
+    buttonText: string;
+    to?: RedirectProps['to'];
+}
+//# sourceMappingURL=PageHeader.interface.d.ts.map
```
Comment (tricaalexandru) [inline dist/components/Organisms/PageHeader/PageHeader.interface.d.ts:12]: @philjonasfiltered do we need this commented out line?

**Review by tricaalexandru — APPROVED** [review]

File: dist/components/Organisms/PageHeader/PageHeader.d.ts (hunk 1 above)
Comment (philjonasfiltered) [reply dist/components/Organisms/PageHeader/PageHeader.d.ts:4]: this line is generated by the build process. It creates a .js/.ts mapping for easier debugging.

File: dist/components/Organisms/PageHeader/PageHeader.interface.d.ts (hunk 2 above)
Comment (philjonasfiltered) [reply dist/components/Organisms/PageHeader/PageHeader.interface.d.ts:12]: same as above 👍

**Review by eTheMan80 — APPROVED** [review]

**Pull request merged**